code/datums/sims.dm

```diff
@@ -442,12 +442,7 @@
 	var/list/motives = list()
 	var/list/datum/simsHolder/simsHolders = list()
 	var/list/datum/simsMotive/simsMotives = list()
-
-#ifdef RP_MODE
-	var/provide_plumbobs = 0
-#else
-	var/provide_plumbobs = 1
-#endif
+	var/provide_plumbobs = 0 // I don't think this is still relevant? Maybe for gimmicks
```

Comment:
relevant for togglesimsmode() proc

Reply:
The issue I kept running into is that when I added hygiene to a classic character, it would also enable plumbobs, hence why I set them to 0, because "sims" as we call them have kind of moved away from its origins, I'd wager.. Unless you mean I should just remove the comment. Also, I can't actually see plumbobs being mentioned in the proc, though maybe it's secret and / or I'm overlooking stuff. Isn't the enabling of plumbobs a toggle for admins separately?